tests/wrappers/utils.py

```diff
@@ -1,7 +1,9 @@
+from __future__ import annotations
+
 import gymnasium as gym
 
 
-def has_wrapper(wrapped_env: gym.Env, wrapper_type: type) -> bool:
+def has_wrapper(wrapped_env: gym.Env, wrapper_type: type[gym.Wrapper]) -> bool:
```

Comment:
Can you double check if the `type[gym.Wrapper]` annotation is actually correct? It seems like it makes sense, but I can't find anything about it right now.

Reply:
I will remove it as it is not necessary for the testing